### DISCUSSION TITLE
Logo permission?

### DISCUSSION THREAD

**Discussion by IvanTheGeek** [AvaloniaUI/Avalonia#13798] [General]
I am rather new to Avalonia. I am researching and learning how to use it with F#. I plan to do some blogging and learning materials to publish (public) to help others (and for my own learning). I am not representing myself as part of the project or team. Is there any problem with using the Avalonia logo in these places?

**Comment by thevortexcloud** [comment]
I would _guess_ likely not. As the logo is included in the source code and the entire project is licensed under MIT. As long as you follow the license there is not much anybody can do to stop you. 

EG the icon file for the MVVM template is here https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/master/templates/csharp/app-mvvm/Assets/avalonia-logo.ico

And [here](https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/master/LICENSE) is the license for that project.

If its intentional that the Avalonia Logo be licensed under MIT I am not sure. I think Avalonia OU (which is the commercial company that most of the main Avalonia devs work for) may also use the same logo? I am not sure. It's technically possible they may have a trade mark for it. Which would be fairly weird if they did. If there is one, there is a lot of other rules you need to legally follow to avoid people confusing your brand with theirs. 

But Some other people may non legal issues with you using it. In any case, to be safe you should probably wait for one one of the main devs to respond.

**Comment by maxkatz6** [comment]
We have Press Kit available on our web-site. You can use materials from there in your content with no issues.
See links in the footer on the https://avaloniaui.net/.
Or direct link (might be changed in the future): https://drive.google.com/uc?id=1laynvVMvQn123I45q4EjexHpDBZcx6rk&export=download

**Reply by IvanTheGeek** [reply]
I never even thought about the press kit! Thank you.

**Comment by IvanTheGeek** [comment]
In the [Unleashing the Power of Cross-Platform Development with Avalonia UI | .NET Conf 2023](https://youtu.be/9PZVjcp3Xxc?si=R0C0Rcb6nG3_4Cu4) video, there is a "OUR ARCHITECTURE" slide. Is that slide or graphic available somewhere (have not been able to find it) or would @MikeCodesDotNET be willing to share that with me?

**Reply by thevortexcloud** [reply]
There is a diagram [here](https://avaloniaui.net/Blog/avalonia-platform-support-why-it-s-simple) that shows the same information. But it's not the same.

**Reply by IvanTheGeek** [reply]
Yes, thank you. I have found that one, but the one in the video is a different view of things and I am hoping to include it as reference.

**Comment by MikeCodesDotNET** [comment]
Is this the one you mean?

![Architecture](https://github.com/AvaloniaUI/Avalonia/assets/552074/ef9a5d1b-ecbc-4fa6-a35d-46154967a022)

To @thevortexcloud, we hold the trademark for "Avalonia" (the name, not the logo). We're generally flexible with its use, but sometimes ask for project name changes to avoid confusion with our official or community projects. And by "we", I mean AvaloniaUI OÜ.

**Reply by thevortexcloud** [reply]
Thanks for clarifying that!

**Reply by IvanTheGeek** [reply]
YES, that is the one! Thank you!
